test/distrib/php/run_distrib_test.sh

```diff
@@ -32,8 +32,9 @@ set -ex
 
 cd $(dirname $0)
 
-cp -r $EXTERNAL_GIT_ROOT/input_artifacts/grpc-php.tgz .
+cp -r $EXTERNAL_GIT_ROOT/input_artifacts/grpc-*.tgz .
```

Comment:
did you check that there isn't a clash between php's artifact name and some other language (e.g. python)?